extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -35,17 +36,37 @@ export class ValidationBubble {
     /** @private @const {!../../../src/service/vsync-impl.Vsync} */
     this.vsync_ = vsyncFor(win);
 
+    /** @private {?Element} */
+    this.currentTargetElement_ = null;
+
+    /** @private {string} */
+    this.currentMessage_ = '';
+
+    /** @private {boolean} */
+    this.isVisible_ = false;
+
     /** @private @const {!HTMLDivElement} */
     this.bubbleElement_ = win.document.createElement('div');
     this.bubbleElement_.classList.add('-amp-validation-bubble');
     this.bubbleElement_[OBJ_PROP] = this;
+    this.bubbleElement_.setAttribute('aria-labeledby',
```

Comment:
Yeah I got this a bit backward. Although this would still work you're right it might point to some non-existing element yet. So will change this.